settings/OpenExceptionalForm/ExceptionWrapper.js

```diff
@@ -1097,7 +1063,30 @@ class ExceptionWrapper extends React.Component {
         defaultWidth="fill"
         paneTitle={paneTitle}
         firstMenu={paneStartMenu}
-        lastMenu={paneLastMenu}
+        footer={(
+          <PaneFooter
+            renderStart={(
+              <Button
+                data-test-cancel-exception-form
+                marginBottom0
+                onClick={() => { this.beforeExit('paneStartMenu'); }}
+              >
+                <FormattedMessage id="ui-calendar.common.cancel" />
+              </Button>
+            )}
+            renderEnd={(
+              <IfPermission perm={permissions.POST}>
+                <Button
+                  buttonStyle="primary"
+                  onClick={() => { this.setState({ openEditor: true }); }}
```

Comment:
Please move the methods props at the bottom and be consistent about the ordering of properties. For example, in render start button data-attribute is in the top and in the render end it is in the bottom.